client/src/App.tsx

```diff
@@ -1,14 +1,15 @@
-import Home from "./pages/Home";
-import {Route, Routes} from "react-router-dom";
+import React from 'react';
+import { Route, Routes} from 'react-router-dom';
+import {Splash} from './pages/splash/Splash';
 
-function App(): JSX.Element {
+const App: React.FC = () => {
   return (
     <div>
       <Routes>
-          <Route path={"/"} element={ <Home/> } />
+        <Route path="/" element={<Splash/>} />
```

Comment:
I can see the splash screen has been set to `Splash` component. After loading the app, it still does not seem to navigate to other pages.
Are you guys having any troubles implementing the splash screen?